README.md

```diff
@@ -97,15 +96,24 @@ const fooInstance = await WebAssembly.instantiate(FooModule, {
 wasi.start(fooInstance);
 ```
 
-#### Imports from WebAssembly
+The static analysis benefits of not needing a custom `fetch` and
+`WebAssembly.compileStreaming` benefit not only code analysis and security but
+also bundlers.
 
-Web Assembly would have the ability to expose import reflection in its
-imports if desired, as the [Module Linking proposal currently aims to specify][module-linking].
+In turn this enables [Wasm components to be able to import][]
+`WebAssembly.Module` objects themselves in future.
 
-#### Security improvements
+### Other Module Types
+
+Other module types may define their own host reflections. Alternatively they may
+throw if there is no suitable reflection available.
```

Comment:
Suggest: a module will fail to link if it depends on a reflected import that the host cannot satisfy for lack of an available reflection for the corresponding module type.

Reply:
Added.